internal/dev_server/ui/src/Flags.tsx

```diff
@@ -149,136 +198,202 @@ function Flags({
+    }
+  };
+
   return (
     <>
-      <div className="container">
-        <Box
-          display="flex"
-          flexDirection="row"
-          justifyContent="space-between"
-          alignItems="center"
-          marginBottom="2rem"
-          padding="1rem"
-          background={Theme.color.blue[50]}
-          borderRadius={Theme.borderRadius.regular}
+      <Box
+        display="flex"
+        flexDirection="row"
+        justifyContent="space-between"
+        alignItems="center"
+        marginBottom="2rem"
+        padding="1rem"
+        background={Theme.color.blue[50]}
+        borderRadius={Theme.borderRadius.regular}
+      >
+        <Label
+          htmlFor="only-show-overrides"
+          className="only-show-overrides-label"
         >
-          <Label
-            htmlFor="only-show-overrides"
-            className="only-show-overrides-label"
-          >
-            <Checkbox
-              id="only-show-overrides"
-              isSelected={onlyShowOverrides}
-              onChange={(newValue) => {
-                setOnlyShowOverrides(newValue);
-              }}
-              isDisabled={!overridesPresent}
-              style={{
-                display: 'inline-block',
-                marginRight: '.25rem',
-              }}
-            />
-            Only show flags with overrides
-          </Label>
-          <Button
-            variant="destructive"
+          <Checkbox
+            id="only-show-overrides"
+            isSelected={onlyShowOverrides}
+            onChange={(newValue) => {
+              setOnlyShowOverrides(newValue);
+            }}
             isDisabled={!overridesPresent}
-            onPress={async () => {
-              // This button is disabled unless overrides are present, but the
-              // type is nullable
-              if (!overrides) {
-                return;
-              }
+            style={{
+              display: 'inline-block',
+              marginRight: '.25rem',
+            }}
+          />
+          Only show flags with overrides
+        </Label>
+        <Button
+          variant="destructive"
+          isDisabled={!overridesPresent}
+          onPress={async () => {
+            // This button is disabled unless overrides are present, but the
+            // type is nullable
+            if (!overrides) {
+              return;
+            }
 
-              const overrideKeys = Object.keys(overrides);
+            const overrideKeys = Object.keys(overrides);
 
-              await Promise.all(
-                overrideKeys.map((flagKey) => {
-                  // Opt out of local state updates since we're bulk-removing
-                  // overrides async
-                  removeOverride(flagKey);
-                }),
-              );
+            await Promise.all(
+              overrideKeys.map((flagKey) => {
+                // Opt out of local state updates since we're bulk-removing
+                // overrides async
```

Comment:
```suggestion
                // Opt out of local state updates since we're bulk-removing
                // overrides async
```

forgot I made this comment irrelevant in another change.